### PR TITLE
Add metric sample codes to README

### PR DESCRIPTION
**Pull request by yong-huang**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -380,6 +382,33 @@ Remove a Resource
...
+    >>> lun1.total_iops
+    0.05
+    >>> lun1.write_iops
+    0
+    >>> lun1.read_byte_rate
+    25.6
+    >>> lun1.write_byte_rate
+    0
+    >>> lun1.total_byte_rate
+    25.6
```
Comment (Murray-LIANG) [inline README.rst]: Add a blank line below.

File: README.rst [hunk 2]
```diff
@@ -380,6 +382,33 @@ Remove a Resource
     # delete a resource
     >>> lun1.delete()
 
+Getting metrics
+```````````````
+
```
Comment (Murray-LIANG) [inline README.rst]: Could you add all supported metrics from metric_configs.yaml? Just list the names.

File: README.rst (hunk 1 above)
Comment (yong-huang) [reply README.rst]: Done.

File: README.rst (hunk 2 above)
Comment (yong-huang) [reply README.rst]: Done.

**Pull request merged**